doc/design/model_zoo.md

```diff
@@ -99,7 +99,11 @@ The training submitter program `my_first_model.py`, running in the model definit
 
 ### Model Zoo Data Schema
 
-The model zoo table is in a database deployed as part of the SQLFlow service. This database might not be the one that holds data sources.  The only requirement of the model zoo table is to have a particular data schema that contains at least the following fields.
+The model zoo table is in a database deployed as part of the SQLFlow service. This database might not be the one that holds data sources.  The only requirement of the model zoo database is to have two tables: the **trained models table** and the **evaluation result table**.
+
+#### Trained Models
+
+Once a training job completes, the submitter program adds/updates a row of the the trained models table, which contains (at least) the following fields.
```

Comment:
Note that if a saved model is large we may have multiple rows, each row stores part of the saved model.

Reply:
How about saving the whole model to storage media like OSS?

Reply:
Maybe for cases like training with PAI Tensorflow, we can store the URI of the saved model file on OSS?